introspection-engine/connectors/sql-introspection-connector/src/introspection_helpers.rs

```diff
@@ -192,11 +169,6 @@ pub(crate) fn calculate_index(index: sql::walkers::IndexWalker<'_>, ctx: &Contex
         Some(index.name().to_owned())
     };
 
-    // We do not populate name in client by default. It increases datamodel noise, and we would
-    // need to sanitize it. Users can give their own names if they want and re-introspection will
-    // keep them. This is a change in introspection behaviour, but due to re-introspection previous
-    // datamodels and clients should keep working as before.
```

Comment:
Deleting this because it was relevant when that was implemented, but it is just what the implementation does now.